server/src/main/kotlin/com/mobilejazz/colloc/util/StringUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.mobilejazz.colloc.util
+
+object StringUtils {
+  object Android {
+
+    private val questionMarkAtStartRegex = "^(\\?)".toRegex()
+    private val referenceAtStartRegex = "^(@)".toRegex()
+    private val tagOpenRegex = "^(<)".toRegex()
+    private val tagCloseRegex = "^(>)".toRegex()
+    internal fun String.correctInvalidLiterals(): String =
+      apply {
+        println("KORT $this")
+      }.replace("%@", "%s")
+        .replace(referenceAtStartRegex, "\\\\@")
+        .replace(questionMarkAtStartRegex, "\\\\?")
+        .replace("'", "\\'")
+        .replace("&", "&amp;")
+        .replace(tagOpenRegex, "&lt;")
+        .replace(tagCloseRegex, "&gt;")
+        .replace("\"", "\\\"")
+        .apply {
+          println("KORT-B $this")
+        }
+  }
```

Comment:
Can we just have it public and with name `String.androidCorrectInvalidLiterals()` and remove the`objec` classes?

Reply:
I would call it something like "encode literals", no literal is invalid and therefore doesn't need to be corrected, just encoded properly.